Dockerfile

```diff
@@ -7,6 +7,18 @@ WORKDIR /usr/src/app
 COPY requirements.txt ./
 RUN pip install --no-cache-dir -r requirements.txt
 
-COPY moj_analytics /usr/local/lib/python3.6/site-packages/moj_analytics
+COPY moj_analytics /usr/local/lib/python3.7/site-packages/moj_analytics
```

Comment:
If we're assuming Docker image has Python 3.7, wouldn't be better to use a more specific tag as a Base image?

(I find the [Python image tags](https://hub.docker.com/r/library/python/tags/) slightly confusing, would be `3.7-alpine` in this case?)

Reply:
yes, i think this change was required due to the base image version being incremented. i will use a more specific base image tag.